src/libtorch.cc

```diff
@@ -1056,6 +1125,15 @@ ModelInstanceState::ProcessRequests(
   std::vector<BackendMemory*> input_memories;
   bool cuda_copy = false;
   std::unique_ptr<BackendInputCollector> collector;
+  if (Kind() == TRITONSERVER_INSTANCEGROUPKIND_GPU) {
+#ifdef TRITON_ENABLE_GPU
+    RESPOND_ALL_AND_SET_TRUE_IF_ERROR(
+        responses, request_count, all_response_failed,
+        ConvertCUDAStatusToTritonError(
+            cudaEventRecord(compute_input_event_, stream_),
```

Comment:
Shouldn't the event be recorded after all input copies are issued?

Reply:
This mainly marks the beginning of the input collection. The input compute part is the duration between `compute_input_event_` and `compute_infer_event_`.

Reply:
after we can use `stream_` for execution with the FromExternal, event is only used to keep track of the duration of different compute phase?

Reply:
Correct.